.vscode/launch.json

```
@@ -30,9 +30,8 @@
             "request": "launch",
             "mode": "debug",
             "program": "${workspaceRoot}/cmd/ddev",
-            "cwd": "${workspaceRoot}/../d9",
             "env": {"DDEV_DEBUG": "true"},
-            "args": ["describe"],
+            "args": ["describe", "sv2"],
```

I'm assuming `sv2` is SvetleKit, but where does it come from? I don't see any `sv2` here.

---

I was using VS Code and experimenting with a project named sv2, it needs to go away :)

```suggestion
            "args": ["describe"],
```